.dockerignore

```diff
@@ -0,0 +1 @@
+.gitignore
```

Comment:
The baseimage is still built when the ubuntu setup script changes, so this is needed.